server/src/main/java/org/elasticsearch/index/mapper/GeoPointFieldMapper.java

```diff
@@ -215,7 +215,33 @@ protected void index(DocumentParserContext context, GeoPoint geometry) throws IO
             context.doc().add(new StoredField(fieldType().name(), geometry.toString()));
         }
         // TODO phase out geohash (which is currently used in the CompletionSuggester)
```

Comment:
what's the plan here? From the code, it does not seem like we can phase out geohash given it's what any geopoint sub-field reads. I believe the comment is inaccurate as also keyword and text will get the geohash.